Tasks/NugetPublisher/nugetpublisher.ts

```diff
@@ -159,6 +162,39 @@ async function main(): Promise<void> {
             var result = Q({});
             for (const packageFile of filesList) {
                 await publishPackageAsync(packageFile, publishOptions);
+                
+                // POST package build metadata to feed service
+                let packageMetadata = await buildMetadataHelpers.getPackageMetadata(packageFile);
+                packageBuildMetadata = {
+                    'PackageName': packageMetadata.Name,
+                    'ProtocolType': 'NuGet',
+                    'BuildId': tl.getVariable('Build.BuildId'),
+                    'CommitId': tl.getVariable('Build.SourceVersion'),
+                    'BuildCollectionId': tl.getVariable('System.CollectionId'),
+                    'BuildProjectId': tl.getVariable('System.TeamProjectId'),
+                    'RepositoryId': tl.getVariable('Build.Repository.Id'), 
+                    'BuildAccountId': null, //NOTE: NULL for on prem, must set this value for hosted
+                    'OriginalPackageVersion': packageMetadata.Version
+                };  
+
+                if (nuGetFeedType == 'internal') {
+                    tl._writeLine('HELLO HELLO HELLO');
+                    var baseUrl = 'http://pkg-styx:8080/tfs/DefaultCollection'; //TODO: this is hard coded
+                    var area = '/_apis/Packaging';
+                    var service = '/Feeds/' + buildMetadataHelpers.getFeedName(internalFeedUri);
```

Comment:
getFeedName could possibly return null. It would be good to handle that with an immediate error, rather than building an invalid URL.